IssueTracker/IssueTracker.FileSystem/Data/IData/IBolbData.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Azure.Storage.Blobs;
+
+namespace IssueTracker.FileSystem.Data.IData;
+public interface IBolbData
```

Comment:
all methods should be async. 
the 'data' suffix is not appropriate for interfaces.

Reply:
![image](https://user-images.githubusercontent.com/74471664/188216361-81263307-b370-4f4c-9942-76b43d864404.png)
all methods are task
About name, IBolbFile? it's ok?